src/Apps/Artwork/Components/ArtworkSidebar/index.tsx

```diff
@@ -53,6 +66,10 @@ export class ArtworkSidebar extends Component<ArtworkSidebarProps> {
           </React.Fragment>
         )}
 
+        <TrustSignalsContainer>
```

Comment:
Can we add a test that verifies that the correct trust signal shows up based on criteria?

Reply:
Can I follow up on this? I want to explore `createTestEnv` and some ways to make setting up these tests a little simpler. Right now the logic is fairly trivial so having a unit test for the actual component itself is likely low value. Ensuring that it renders in the sidebar on expected works _is_ a test that I want to implement for all of these though.

Reply:
👍 agreed on the higher-value sidebar tests.